Makefile

```diff
@@ -1,36 +1,14 @@
-# HURUmap Makefile
-# ----------------
-
-# HURUmap base
-
-base:
-	docker build --no-cache -t codeforafrica/hurumap-base:0.1.2 contrib/base
-	docker build -t codeforafrica/hurumap-base:latest contrib/base
-	docker push codeforafrica/hurumap-base:0.1.2
-	docker push codeforafrica/hurumap-base:latest
-
-
-# HURUmap app
+COMPOSE = docker-compose
+DEVDOCKER = $(COMPOSE) exec web
 
 build:
-	docker-compose build
-
-image-latest:
-	docker build --no-cache -t codeforafrica/hurumap:latest .
-	docker push codeforafrica/hurumap:latest
-
-image-release:
-	docker build --no-cache -t codeforafrica/hurumap:1.0.1
-	docker push codeforafrica/hurumap:1.0.1
-
-
-# Development
+	$(COMPOSE) build
 
 web:
 	rm -fr static/*  # Workaround for whitenoise busyness in dev
```

Comment:
`rm -fr static/*` always fails for me (I think because the files are created by `docker` which always run in `sudo` mode).

Can't we use `--clear` in `collectstatic` to ensure files are removed before creating/linking new ones? (We may need to use `DEBUG` in `entrypoint.sh` to determine if we should pass the `--clear` in development and skip it in production.)